terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-up-chron.test.ts

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment';
+import nock from 'nock';
+import { mocked } from 'ts-jest/utils';
+import { Config } from './config';
+
+// Import the required modules
+import { getQueuedJobs } from './scale-up-chron';
+
+
+describe('scaleUp', () => {
+
+  it('getQueuedRunners should fetch queued runners', async () => {
```

Comment:
We probably want to test a bit more here, check a few basics: mock `getQueuedJobs` and see if `scaleUp` is properly being called, if the filtering is working, how we're handling exceptions, etc.

Reply:
Ideally we would like to see the full behaviour of the function being tested. This does not mean that we need to cover 100% of lines/branch/statements, but we do want to see all its functionally tested.